litex/soc/integration/soc.py

```diff
@@ -1814,6 +1822,12 @@ def add_etherbone(self, name="etherbone", phy=None, phy_cd="eth", data_width=8,
         from liteeth.frontend.etherbone import LiteEthEtherbone
         from liteeth.phy.model import LiteEthPHYModel
 
+        # Parameters.
```

Comment:
We should probably move these checks directly to the `with_ethmac `section.

Reply:
done